webview/src/experiments/components/table/styles.module.scss

```diff
@@ -237,404 +233,389 @@ $bullet-size: calc(var(--design-unit) * 4px);
   height: 100%;
   display: flex;
   flex-flow: column nowrap;
+}
```

Comment:
Looking at the styles, the main issues seem to be the fact that there is a lot of nesting and classes are selected at different parts throughout the file, making it difficult to actually see what styles are being applied to what. 

I figured the first step is moving everything out of experiments then start grouping the classes better.